### ISSUE TITLE
Multi-literal strings support

### ISSUE THREAD

**Issue by k06a** [duaraghav8/Ethlint#290]
**Description**

We need support for multi-literal strings like this:
```solidity
bytes32 constant public HUMAN_TYPEHASH = keccak256(
    "Human("
    "string name,"
    "string surname,"
    "string street,"
    "address wallet,"
    "uint256 age,"
    "uint256 birth"
    ")"
);
```

Here is demo repo: https://github.com/k06a/truffle-string-multi-literal

Check [Solidity Grammar](https://docs.soliditylang.org/en/v0.8.2/grammar.html#a4.Solidity.stringLiteral):
<img width="610" alt="image" src="https://user-images.githubusercontent.com/702124/111865568-27aa2c00-8968-11eb-9bb5-3f5397a2afa6.png">

**Steps to reproduce**

1. `git clone "https://github.com/k06a/truffle-string-multi-literal"`
2. `cd truffle-string-multi-literal`
3. `npm install` or `yarn`
4. `npm lint` or `yarn lint`

**Expected behavior**

This is valid Solidity code.

**Operating System**

macOS 10.11.2

**Linter version**

Solium version 1.2.5
